inc/blocks/namespace.php

```diff
@@ -131,7 +181,7 @@ function on_save_post( int $post_ID, WP_Post $post, bool $update ) : void {
 				'post_name' => $xb['attrs']['clientId'],
 				'post_author' => get_current_user_id(),
 				'post_title' => $xb['attrs']['title'] ?: $default_title,
-				'post_parent' => $post_ID,
+				'post_parent' => $post_ID ?: '',
```

Comment:
The field type in the database is an integer so if there's no parent use `null` or `0` as the fallback. This is why setting a default value in the parameters is sometimes useful.